lib/MusicBrainz/Server/Entity/LinkAttribute.pm

```diff
@@ -25,7 +25,7 @@ sub html {
     my $type = $self->type;
     my $value = encode_entities($type->l_name);
 
-    if ($type->root->id == $INSTRUMENT_ROOT_ID && $type->gid) {
+    if (defined $type->root->id && $type->root->id == $INSTRUMENT_ROOT_ID && $type->gid) {
```

Comment:
This one seems odd to me too, since if we were able to load `$type->root` I'd think we should have the id...hmm.